go-controller/pkg/metrics/master.go

```diff
@@ -143,6 +143,29 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+var MetricNetworkPolicyCountInNameSpace = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
We can probably get rid of this - I just realized this is already exported from kube-state-metrics -- https://github.com/kubernetes/kube-state-metrics/blob/5d56275d2b95ffcf6d15d8a6fa2b1bb02a1066ae/internal/store/networkpolicy.go

Reply:
Please let me know if I should close this PR .

Reply:
no, just remove this metric.

Reply:
Done